colcon_core/task/python/test/pytest.py

```diff
@@ -67,8 +74,14 @@ def match(self, context, env, setup_py_data):  # noqa: D102
                     *(Path(context.args.build_base).parts)) / 'coverage.html'),
                 '--cov-report=xml:' + str(PurePosixPath(
                     *(Path(context.args.build_base).parts)) / 'coverage.xml'),
-                '--cov-branch',
             ]
+            # use --cov-branch option only when available
+            # https://github.com/pytest-dev/pytest-cov/blob/v2.5.0/CHANGELOG.rst
+            from pytest import __version__ as pytest_cov_version
```

Comment:
Should be `from pytest_cov import ...`

Reply:
Good catch! Fixed in d928c102bd0b110982ba494eeecbf4e2f20619ff. Please re-review.